src/cmd/raw.go

```diff
@@ -0,0 +1,84 @@
+package cmd
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/tkhq/go-sdk/pkg/api/client/private_keys"
+	"github.com/tkhq/go-sdk/pkg/api/models"
+)
+
+var (
+	rawSignPayloadEncoding string
+	rawSignHashFunction    string
+	rawSignPayload         string
+)
+
+func init() {
+	rawCmd.Flags().StringVar(&signingKeyID, "signing-key", "", "name or ID of the signing key")
```

Comment:
We call this "Private Key" everywhere in our docs, so let's stick with that wording here as well?

Reply:
Unfortunate choice of terms, to my mind, but consistency is important.